README.md

```diff
@@ -602,6 +605,186 @@ Response Body/Total Sizes : count 40 avg 12565.2 +/- 301.9 min 12319 max 13665 s
 All done 40 calls (plus 4 warmup) 60.588 ms avg, 7.9 qps
 ```
 
+
+### Remote triggered load test (server mode rest API)
+
+New since 1.18 the server has a `fortio/rest/run` endpoint similar to what the form UI submit in `fortio/` to start a run.
+  - plus `async` query arg or json value `"on"` will make the run asynchronous (returns just the runid of the run instead of waiting for the result)
+  - plus read all the run configuration from either query args or jsonPath POSTed info
+  - compatible with [flagger](https://github.com/fluxcd/flagger) and other webhooks
+  - New in 1.22: use `headers` json array to send headers (or multiple `&H=` query args)
```

Comment:
maybe add a comment, that if the same header is passed in both places, `headers` will overwrite &H

Reply:
good point, for everything else the command line takes precedence so maybe I should flip the order for consistency